### ISSUE TITLE
Python version

### ISSUE THREAD

**Issue by BitSecret** [tongxiao2002/DualGeoSolver#2]
What is the python version of this project?

**Comment by Jayavardhan-7** [comment]
# Do you me the python version or specific package version.
## For python version :- 
* run this code in cmd:
* python --version

**Comment by BitSecret** [comment]
Yes, the Python version. Can you tell me the Python version of the DualGeoSolver project on your computer? I want to reproduce your code.







---- Replied Message ----
| From | ***@***.***> |
| Date | 9/17/2024 20:47 |
| To | ***@***.***> |
| Cc | ***@***.***>,
***@***.***> |
| Subject | Re: [tongxiao2002/DualGeoSolver] Python version (Issue #2) |
Do you me the python version or specific package version.
For python version :-
run this code in cmd:
python --version

**Comment by tongxiao2002** [comment]
Python 3.8. Thank you for the reminder, we will add it to the README.md.

**Comment by BitSecret** [comment]
Thank you for your reply. May I ask what the version of the `transformers` library is?

**Comment by tongxiao2002** [comment]
> Thank you for your reply. May I ask what the version of the `transformers` library is?

transformers==4.17.0, as stated in `requirements.txt`
